main.go

```diff
@@ -82,6 +83,7 @@ func main() {
 	flag.BoolVar(&watchAllNamespaces, "watch-all-namespaces", true,
 		"Watch for custom resources in all namespaces, if set to false it will only watch the runtime namespace.")
 	flag.DurationVar(&rateLimitInterval, "rate-limit-interval", 5*time.Minute, "Interval in which rate limit has effect.")
+	flag.BoolVar(&httpSchemeDisabled, "http-scheme-enabled", false, "Enable Http Scheme When true, the flag would not allow the use of the http scheme across all controller-level objects.")
```

Comment:
We could name it in a way that it portrays this is not the secure thing to do:
```suggestion
	flag.BoolVar(&insecureNoTLS, "insecure-no-tls", false, "Enable the use of the HTTP scheme (no TLS) across all controller-level objects. This is not recommended for production environments. ")
```